src/hotspot/share/opto/library_call.cpp

```diff
@@ -4945,6 +4946,57 @@ bool LibraryCallKit::inline_unsafe_copyMemory() {
   return true;
 }
 
+bool LibraryCallKit::inline_unsafe_setMemory() {
+  if (callee()->is_static())  return false;  // caller must have the capability!
```

Comment:
Also need to return false if StubRoutines::unsafe_setmemory() == nullptr.

Reply:
Will do.